docs/advanced-topics/linear-probe.md

```diff
@@ -0,0 +1,61 @@
+(linear-probe)=
+# {octicon}`pin` Linear Probe
+
+## Why freezing?
+
+Depending on your task and the amount of training data,
+sometimes it is unnecessary to tune the entire model.
+In these cases,
+freezing the weights of the pre-trained model and fine-tuning specific layers is sufficient enough.
+Besides, freezing can reduce the training time considerably.
+
+Finetuner allows you to fine-tune a Linear Probe (or Projection Head) easily.
+
+```{warning}
+Currently, we only allow you to freeze layers for image-to-image search tasks.
+These models are built on top of Convolution Neural Networks (CNNs).
+
+For transformer architecture,
+we always train the entire Neural Nets.
+If this feature is needed, consider submitting a feature request in our [Github Issues page](https://github.com/jina-ai/finetuner/issues)
+```
+
+## How?
+
+Finetuner has a built-in module called Tailor.
+Given a general model written in Pytorch,
+Tailor performs micro-operations on the model architecture and outputs an embedding model.
+
+Given a general model with weights, Tailor preserves its weights and performs (some of) the following steps:
+
++ Finding all dense layers by iterating over layers.
++ Chopping off all layers after a certain dense layer.
++ Freezing weights of specific layers.
++ Adding a new module on top of the embedding model.
+
+![tailor](../imgs/tailor.svg)
+
+With Finetuner, you can simply put `freeze=True` and `output_dim=X` like this:
```

Comment:
For example, just using the arguments `freeze=True` and `output_dim=X` with the `fit` function, as shown below: